README.md

```diff
@@ -1,104 +1,151 @@
+
+Changes to the application's repository, update to the base image or any RPMs that are part of the image will trigger a new build
+
+3. **Building the image**
+
+The container image is built within OpenShift and then pushed to OpenShift's internal registry. This results in an image tagged `image:test`.
 
 4. **Test the application**
 
-     Can be a script (mentioned in the [yaml file](https://github.com/CentOS/container-index)) which runs tests on above image. Result: image tagged with a [hash based on date & time](https://github.com/CentOS/container-pipeline-service/blob/master/jenkinsbuilder/project-defaults.yml#L20) pushed to internal registry.
- 
-5. **Scan the image**
-    
-    Scan uses [atomic scan](https://github.com/projectatomic/atomic) tooling. Multiple atomic scanners are run on the built image and different checks are done - check if image has outdated RPM, npm, pip, gem packages and if image has tampered files present, etc. More details about the scanners can be found in `atomic_scanners` directory of this repo. Result: image tagged as `:rc` pushed to internal registry.
-    
-6. **Deliver to public registry**
+Tests as well as any scripts may be specified within the `yaml` file. For example, the `JenkinsBuilder` image uses a test to "tag" the resulting image (https://github.com/CentOS/container-pipeline-service/blob/master/jenkinsbuilder/project-defaults.yml#L20).
 
-    A simple script to re-tag image to it's final name based on value in the yaml file on Container Index. Result: image tagged with `:<desired_tag>` pushed to https://registry.centos.org. You can refer to [Container Pipeline Wiki page](https://wiki.centos.org/ContainerPipeline) to find currently available container images. This page is automatically updated when a new image is built in the Pipeline.
+5. **Scan and analyze the image**
 
-7. **Email to the Developer**
+Scanning and analysis is done by using [Atomic](https://github.com/projectatomic/atomic).
 
-    An email is sent out the developer mentioning the status of the lint, build and scan processes and a link (s)he can use to read the detailed logs.
- 
-All the communication between the stages mentioned above happens via [beanstalkd](http://kr.github.io/beanstalkd/) tubes.   
+Multiple [Atomic Scanners](/atomic_scanners) are ran on the built image:
 
-## Contribute to Container Pipeline Service
+- [pipeline-scanner](atomic_scanners/pipeline-scanner): Scans for any outdated yum packages
+- [container-capabilities-scanner](atomic_scanners/container-capabilities-scanne): Scans for any security vulnerabitlies in the `RUN` command
+- [atomic_scanners](atomic_scanners/misc-package-updates): Scans for outdated npm, pip and gem packages
 
-We're always looking for ideas and improvements for the service! If you're interested in contributing to this repository, follow these simple steps:
+After scanning and analysis, the resulting image is tagged and pushed to the internal registry as `image:rc`
```

Comment:
Again, no internal registry is used. We push to public registry with a generated test tag.